README.md

```diff
@@ -3,21 +3,20 @@ gist-doc
 
 This repository contains documentation of the [Semantic Arts gist minimalist upper ontology](https://github.com/semanticarts/gist/tree/master) in three formats: narrative, graphical, and Widoco-style.
 
-Current version of gist: 11.0
+Current version of gist: 12.0.0
 
 Contents
 -----
 
-* **gist Constellations (eBook)**
-  * The organization of the eBook is based around the idea of 'constellations' of classes and predicates that deal with related concepts.  
-  * epub and mobi formats
-* **gist Constellations (visualizations)**
-  * A collection of images built using the [Turtle Editor Viewer](http://semantechs.co.uk/turtle-editor-viewer/).  Each image is made up of a single 'constellation' of classes as described above and shows the classes and their relationships, the literal annotations, and any anonymous classes used in class definitions.
+* **gist eBook**
```

Comment:
```suggestion
* **gist eBook: _The Zest of Gist_**
```
Not sure if this formatting works. I'm going for the title in italics inside the bold.